### PR TITLE
modify DrawerReportPage & ModalDrawerReportPage verbiage structure

### PR DESCRIPTION
**Pull request by braxex**
## Description

Needed changes to verbiage structure for DrawerReportPage and ModalDrawerReportPage. Had to add some types and modified some tests as part of that as well.

### How to test

n/a

### Changed Dependencies

n/a

## Code author checklist
- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://bit.ly/3zPrxuZ) tests
- [x] ~~I have added analytics, if necessary~~
- [x] ~~I have updated the documentation, if necessary~~

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

File: services/ui-src/src/components/fields/DropdownField.tsx [hunk 1]
```diff
@@ -64,13 +64,12 @@ export const DropdownField = ({
...
           value: option.id,
         }));
-
         dropdownOptions = fieldOptions;
       }
     } else {
       dropdownOptions = options;
     }
-    if (dropdownOptions[0].value !== "") {
+    if (dropdownOptions[0]?.value !== "") {
```
Comment (braxex) [inline services/ui-src/src/components/fields/DropdownField.tsx:72]: Note: Necessary to fix a bug.

**Review by gmrabian — APPROVED** [review]
lgtm locally

File: services/ui-src/src/forms/mcpar/mcpar.json [hunk 2]
```diff
@@ -973,16 +973,23 @@
...
+                "dashboardTitle": "Add measures for monitoring access standards",
                 "addEntityButtonText": "Add access measure",
                 "editEntityButtonText": "Edit measure",
-                "deleteEntityButtonAltText": "Delete measure"
+                "addEditModalAddTitle": "Add access measure",
+                "addEditModalEditTitle": "Edit access measure",
+                "addEditModalMessage": "Complete the remaining indicators for this access measure by saving the measure and selecting \"Enter measure details\".",
+                "deleteEntityButtonAltText": "Delete measure",
+                "deleteModalTitle": "Delete access measure?",
+                "deleteModalConfirmButtonText": "Yes, Delete Measure",
```
Comment (britt-mo) [inline services/ui-src/src/forms/mcpar/mcpar.json:985]: nit: are these all supposed to be capitalized?

**Review by britt-mo — APPROVED** [review]
looks good, left one comment

File: services/ui-src/src/forms/mcpar/mcpar.json (hunk 2 above)
Comment (braxex) [reply services/ui-src/src/forms/mcpar/mcpar.json:985]: Ooh good catch. The capitalization on line 985 diverges from the designs. Changed.

**Pull request merged**